components/dataComponents/contactdata.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React from 'react'
+
+export default function ContactData({
+  linkType,
+  text,
+  imagePath,
+}: {
+  linkType: string
+  text: string
+  imagePath: string
+}) {
+  let Type: string
+  if (linkType === 'address') {
+    Type = `https://www.google.co.il/maps/search/` + `${text}`
+  } else {
+    Type = linkType === 'tel' ? 'tel:' : 'mailto'
+  }
+  return (
+    <a
+      type={linkType}
+      href={`${Type}`}
```

Comment:
you can just put href={type}

Reply:
fixed
href={`${Type}`} changed to => href={type}